### ISSUE TITLE
Midline brainstem cleft

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6714]
**Preferred term label:**
Midline brainstem cleft

**Synonyms**

**Definition (free text, please give PubMed ID)**

A developmental defect characterized by an abnormal cleft (V-shaped indentation of the stalklike part of the brain consisting of the medulla oblongata, the midbrain, and the pons. 
PMID: 19933510
PMID: 32779696

**Parent term (use hpo.jax.org/app)**
Abnormality of brainstem morphology HP:0002363

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 617542 , Gaze palsy, familial horizontal, with progressive scoliosis, 2
3/3 PMID:28250456

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
